data/js/about-home.js

```diff
@@ -36,12 +36,7 @@ function showSnippet(snippetContent) {
 self.port.on('modify', function(snippetContent) {
     // see whether a default snippet container exists
     if (defaultSnippetContainer) {
-        // try to grab the stuff we're going to insert into the page
```

Comment:
@FrancescoSTL @schalkneethling we need to test this to ensure this isn't going to duplicate content inside of the funnelcake.